test/testHeadless.m

```diff
@@ -35,6 +36,7 @@
         makeTravisReport(status);
     end
 
+    restoreGlobalState(state);
```

Comment:
We probably want to go with the `@onclose` approach. I need to look it up again. @egeerardyn, if you agree that this is the correct way of doing this for a first step, please add a short comment. (Things might change with #575)